api-server/src/models/mutations/job-ad.ts

```diff
@@ -165,6 +166,28 @@ export async function createJobAd(
 	return execTransactionRW(transaction);
 }
 
+function formatWorkExperiences(workExperiences: any): any {
+	// prettier-ignore
+	workExperiences?.forEach(function(_: any, index: number) {	
```

Comment:
I guess we should upgrade prettier to a version that supports the optional chaining operator.

Reply:
We shouldn't do this now though. This PR is big enough as is.